packages/o365/data_stream/audit/manifest.yml

```
@@ -112,49 +112,49 @@ streams:
       - name: resource_retry_max_attempts
         type: text
         title: Resource Retry Max Attempts
-        description: The maximum number of retries for the HTTP client. Default is "5".
+        description: Maximum number of retries for the HTTP client. Default is "5".
         show_user: false
         multi: false
         required: false
       - name: resource_retry_wait_min
         type: text
         title: Resource Retry Wait Min
-        description: The minimum time to wait before a retry is attempted. Default is "1s".
+        description: Minimum time to wait before a retry is attempted. Default is "1s".
         show_user: false
         multi: false
         required: false
       - name: resource_retry_wait_max
         type: text
         title: Resource Retry Wait Max
-        description: The maximum time to wait before a retry is attempted. Default is "60s".
+        description: Maximum time to wait before a retry is attempted. Default is "60s".
         show_user: false
         multi: false
         required: false
       - name: resource_redirect_forward_headers
         type: bool
         title: Resource Redirect Forward Headers
-        description: When set to true resource headers are forwarded in case of a redirect. Default is "false".
+        description: If set to true resource headers are forwarded in case of a redirect. Default is "false".
         show_user: false
         multi: false
         required: false
       - name: resource_redirect_headers_ban_list
         type: text
         title: Resource Redirect Headers Ban List
-        description: When Redirect Forward Headers is set to true, all headers except the ones defined in this list will be forwarded. All headers are forwarded by default.
+        description: If Redirect Forward Headers are set to true, all headers except the ones defined in this list will be forwarded. All headers are forwarded by default.
```

```suggestion
        description: If "Resource Redirect Forward Headers" is set to true, all headers except the ones defined in this list will be forwarded. All headers are forwarded by default.
```
The thing here is singular.